src/components/utils/directory-content-utils.ts

```diff
@@ -17,16 +17,19 @@ import { UserCellRenderer } from './renderers/user-cell-renderer';
 import { DateCellRenderer } from './renderers/date-cell-renderer';
 import type { ElementAttributes } from '@gridsuite/commons-ui';
 
-export const formatMetadata = (data: ElementAttributes, childrenMetadata: Record<UUID, ElementAttributes>) => ({
+export const formatMetadata = (
+    data: ElementAttributes,
+    childrenMetadata: Record<UUID, ElementAttributes>
+): ElementAttributes => ({
     ...data,
-    subtype: childrenMetadata[data.elementUuid]?.specificMetadata.type,
+    subtype: childrenMetadata[data.elementUuid]?.specificMetadata.type as unknown as string,
```

Comment:
don't cast like that if the type in commons-ui isn't correct
```suggestion
    // @ts-expect-error TODO: "Type `object` is not assignable to type `string`"
    subtype: childrenMetadata[data.elementUuid]?.specificMetadata.type,
```

Reply:
Done